graphium/cli/train_finetune.py

```diff
@@ -38,6 +51,18 @@ def run_training_finetuning(cfg: DictConfig) -> None:
 
     cfg = OmegaConf.to_container(cfg, resolve=True)
 
+    dst_dir = cfg["constants"].get("results_dir")
+    hydra_cfg = HydraConfig.get()
+    output_dir = hydra_cfg["runtime"]["output_dir"]
+
+    if dst_dir is not None and fs.exists(dst_dir) and len(fs.get_mapper(dst_dir).fs.ls(dst_dir)) > 0:
+        logger.warning(
+            "The destination directory is not empty. "
+            "If files already exist, this would lead to a crash at the end of training."
```

Comment:
Because you're not sure it will crash. The exact filenames that will be saved to the destination directory depend on the configuration and there is no easy way to figure them out. Similar to how the PTL `ModelCheckpoint` callback also just [warns](https://lightning.ai/docs/pytorch/stable/_modules/lightning/pytorch/callbacks/model_checkpoint.html#ModelCheckpoint.setup) you when the directory is not empty.